test/ventilation_test.c

```diff
@@ -29,16 +32,60 @@ int startVentilation_afterOneInspiration_openInspiratoryValve() {
 
   //reinitialize the system to be sure that the valve wasn't previously open
   close_inspiratory_valve();
+  reset_to_inspiration_start();
   
   start_inspiration();
 
   int inspiratory_valve_status = get_inspiratory_valve_status();
   assertTrue(inspiratory_valve_status == VALVE_OPEN);
 }
 
+int startVentilation_afterOneInspiration_setsVentilationPhaseToInspiration() {
+  //initialize the function get_current_time
+  int* times = (int*) malloc(1*sizeof(int)); 
+  times[0] = 260;
+  set_current_time(times, 1);
+
+  //initialize the function get_respiratory_rate
+  uint32_t* RR = (uint32_t*) malloc(1*sizeof(uint32_t)); 
+  RR[0] = 7500000;
+  status_t* status= (status_t*) malloc(1*sizeof(status_t));
+  status[0] = STATUS_OK;
+  set_respiratory_rate(RR, status, 1);
+
+  reset_to_inspiration_start();
+
+  start_inspiration();
```

Comment:
`start_inspiration` should have a function doc string describing its behaviour. I can see this in `ventilation.h` but what is there is not enough to make sense of how it should behave under test.

Reply:
the definition is in SRS-0010 do you think we should duplicate the information here ?

Reply:
Most of the case to go in DSs arer outside the scope
The rest of the imformation is more about implementation and what the code does at a low level